MAUI/Cartesian-Charts/Legend.md

```diff
@@ -324,6 +365,23 @@ this.Content = chart;
 
 ![Legend layout for cartesian chart](Legend-images/cartesian_chart.png)
 
+## Event 
+
+**LegendItemCreated**
+
+The [`LegendItemCreated`]() event is triggered when the chart legend item is created. This argument contains the following information.
+
+* [`LegendItem`]() – Used to customize the label and appearance of individual legend item. 
```

Comment:
The text color, font size, icon brush, etc., are obtained from the legend item or chart legend.

If the legend item is used, why is there a need to explain the legend item and other customizable properties in this way?
<img width="341" alt="image" src="https://github.com/user-attachments/assets/f7dec7aa-a678-4338-8844-22071fac61cc">

refer this https://help.syncfusion.com/maui/cartesian-charts/axis/types#labelcreated

Reply:
updated content